artman/tasks/publish/github.py

```diff
@@ -55,13 +57,14 @@ def execute(self, git_repo, api_name, api_version, language,
         # ultimately mutate the working directory.
         original_directory = os.curdir
 
-        # Make the GAPIC code directory an absolute path, since we will be
-        # moving around.
-        gapic_code_dir = os.path.realpath(gapic_code_dir)
+        # Track our code directories, and use absolute paths, since we will
+        # be moving around.
+        code_dirs = {'gapic': os.path.abspath(gapic_code_dir)}
```

Comment:
You plan to replace gapic_code_dir and grpc_code_dir with paths in a follow-up PR?